coroutines-interop/src/commonMain/kotlin/com/badoo/reaktive/coroutinesinterop/SchedulerAsCorotuineDispatcher.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.reaktive.coroutinesinterop
+
+import com.badoo.reaktive.scheduler.Scheduler
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.InternalCoroutinesApi
+
+@ExperimentalCoroutinesApi
+@InternalCoroutinesApi
```

Comment:
Why `InternalCoroutinesApi`?

Reply:
Removed internal API, now delays are processed automatically via Default dispatcher. Basically implementing `Delay` interface allows you to handle delays "natively".